<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefaultEditTab</class>
 <widget class="QWidget" name="DefaultEditTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Queue</string>
       </property>
       <property name="buddy">
        <cstring>combo_queueInterfaces</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&amp;Optimizer</string>
       </property>
       <property name="buddy">
        <cstring>combo_optimizers</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>&amp;Template</string>
       </property>
       <property name="buddy">
        <cstring>combo_templates</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QListWidget" name="list_optStep"/>
     </item>
     <item row="4" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="push_add">
         <property name="text">
          <string>Add..</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="push_remove">
         <property name="text">
          <string>Remove...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="push_help">
         <property name="text">
          <string>&amp;Help</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="push_saveScheme">
         <property name="text">
          <string>&amp;Save Opt Scheme</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="push_loadScheme">
         <property name="text">
          <string>&amp;Load Opt Scheme</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>user1:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>user2:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>user3:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>user4:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_user2"/>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_user1"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="combo_queueInterfaces"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combo_optimizers"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="combo_templates"/>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_user4"/>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_user3"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="push_queueInterfaceConfig">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Configure...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="push_optimizerConfig">
       <property name="text">
        <string>Configure...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="edit_edit"/>
   </item>
   <item>
    <widget class="QListWidget" name="list_edit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>list_optStep</tabstop>
  <tabstop>edit_user2</tabstop>
  <tabstop>edit_user4</tabstop>
  <tabstop>edit_edit</tabstop>
  <tabstop>list_edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
